src/views/Admin/AdminEditPermissions.js

```diff
@@ -43,24 +45,24 @@ export default function AdminEditPermissions() {
   const { pointCursor } = usePointCursor();
   const { getDetails } = usePointCache();
 
+  const _contracts = need.contracts(contracts);
+
   const [rekeyDate, setRekeyDate] = useState(Nothing());
 
   const point = need.point(pointCursor);
+  const pointSize = azimuth.getPointSize(point);
+  const isParent = pointSize !== azimuth.PointSize.Planet;
+  const isSenate = pointSize === azimuth.PointSize.Galaxy;
   const userAddress = need.wallet(wallet).address;
 
   const details = need.details(getDetails(point));
-  const { canManage, canTransfer, canSpawn, canVote } = usePermissionsForPoint(
-    userAddress,
-    point
-  );
+  const { canManage, isOwner } = usePermissionsForPoint(userAddress, point);
 
   useLifecycle(() => {
-    getRekeyDate(need.web3(web3), need.contracts(contracts), point).then(
-      res => {
-        if (res === null) setRekeyDate(Just(new Date()));
-        else setRekeyDate(Just(res));
-      }
-    );
+    getRekeyDate(need.web3(web3), _contracts, point).then(res => {
```

Comment:
since we're `need`ing contracts and point we might as well need.web3 outside of this block as well